user_tools/src/spark_rapids_dataproc_tools/dataproc_utils.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
```

Comment:
update copyright to 2022-2023 (applicable for other files as well)

Reply:
Done!
This is the only file that is from 2022.